recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
 {% set name = "perspective" %}
 {% set pypi_name = "perspective-python" %}
-{% set version = "0.9.0" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
-  sha256: fd793cc1070ee21a089ff43d916242a55e512d4f7d8fb2b61c1174f902fd7942
+  # url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  url: https://github.com/finos/perspective/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4050ff749784f4ea77072fbfe8eaf53854d3671c75597a911ead657c77e1d2e5
```

Comment:
Ping @conda-forge/perspective! I had to use the GitHub url here as it appears the sdist wasn't published.
* https://pypi.org/project/perspective-python/0.10.0/#files